client/src/components/pages/admin/DataVisualizationPage.js

```diff
@@ -0,0 +1,373 @@
+                                                      <tr>
+                                                        <td>
+                                                          <p><i className="fa fa-square aero"></i>Symbian </p>
+                                                        </td>
+                                                        <td>15%</td>
+                                                      </tr>
+                                                      <tr>
+                                                        <td>
+                                                          <p><i className="fa fa-square red"></i>Others </p>
+                                                        </td>
+                                                        <td>30%</td>
+                                                      </tr>
+                                                </table>
+                                            </td>
+                                        </tr>
+                                    </table>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div className="col-md-4 col-sm-4 col-xs-12">
+                            <div className="x_panel tile fixed_height_320">
+                                <div className="x_title">
+                                    <h2>Quick Settings</h2>
+                                    <ul className="nav navbar-right panel_toolbox">
+                                        <li><a className="collapse-link"><i className="fa fa-chevron-up"></i></a>
+                                            </li>
+                                            <li className="dropdown">
+                                              <a href="#" className="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><i className="fa fa-wrench"></i></a>
+                                              <ul className="dropdown-menu" role="menu">
+                                                <li><a href="#">Settings 1</a>
+                                                </li>
+                                                <li><a href="#">Settings 2</a>
+                                                </li>
+                                              </ul>
+                                            </li>
+                                            <li><a className="close-link"><i className="fa fa-close"></i></a>
+                                            </li>
+                                    </ul>
+                                    <div className="clearfix"></div>
+                                </div>
+                                <div className="x_content">
+                                    
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+
+            );
```

Comment:
As it stands, this is way too much JSX HTML for this render method. This should be broken up into many sub components, but first I think it should be refactored to use [this](http://www.reactd3.org/).